File: SignatureRequests/SignatureRequests.Webs/Scripts/react/src/Entities/UserRequest.ts
import SignatureEntity from "./SignatureEntity";

class UserRequest {
  role: string;
  name: string;
  email: string;
  password: string;
  signature?: SignatureEntity;
  initial?: SignatureEntity;

  constructor(data: any) {
    this.role = data.Role;
    this.name = data.Name;
    this.email = data.Email;
    this.password = data.Password;
    this.signature = data.Signature;
    this.initial = data.Initial;
  }
}

export default UserRequest;

File: SignatureRequests/SignatureRequests.Core/RequestObjects/UserRequest.cs
﻿using BTSuggestions.Core.Entities;
using SignatureRequests.Core.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SignatureRequests.Core.RequestObjects
{
    public class UserRequest
    {
        public int Id { get; set; }
        public string Role { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public SignatureEntity Signature { get; set; }
        public int? SignatureId { get; set; }
        public SignatureEntity Initial { get; set; }
        public int? InitialId { get; set; }
    }
}
